evap/results/exporters.py

```diff
@@ -60,15 +61,20 @@ def init_styles(self, workbook):
             palette_index = self.CUSTOM_COLOR_START + self.NUM_GRADE_COLORS + i
             style_name = self.deviation_to_style(deviation)
             self.create_style(workbook, deviation_base_style, style_name, palette_index, color)
+            self.create_style(workbook, deviation_base_style + ', left medium', style_name + '_total', palette_index, color)
```

Comment:
4.000 styles are allowed per document. so we should be fine.
i will change the color definitions.